scripts/main.js

```diff
@@ -39,8 +39,38 @@ let question_index = 0;
 
 const wrapContainer = document.querySelector('#wrap');
 
-let quizCard = document.createElement('div');
+let quizCard = document.createElement('form');
 quizCard.className="quiz";
+quizCard.nam = "answer";
+quizCard.addEventListener("submit", (e)=>{
+    e.preventDefault();
+    const formData = new FormData(quizCard);
+    const checkedRadio = formData.get('answer');

```

Comment:
There is no neeed in formData. It should look like this
```suggestion
    const checkedRadio = e.target.answer.value

```

Reply:
Ah, okay. I thought with the target there could be some 
event bubbling problem. I will fix it now.